### ISSUE TITLE
Homebrew installation errors: `Failed changing dylib ID` and `Failed to fix install linkage`

### ISSUE THREAD

**Issue by treykasada** [ruimarinho/gsts#40]
## Summary
Yo! I just today attempted to install `gsts` via Homebrew and got some interesting errors:

```
treykasada@MacBook-Pro ~> brew tap ruimarinho/tap
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 2 taps (homebrew/core and homebrew/cask).
==> Updated Formulae
Updated 13 formulae.
==> Updated Casks
modern-csv

==> Tapping ruimarinho/tap
Cloning into '/usr/local/Homebrew/Library/Taps/ruimarinho/homebrew-tap'...
remote: Enumerating objects: 5, done.
remote: Counting objects: 100% (5/5), done.
remote: Compressing objects: 100% (4/4), done.
remote: Total 5 (delta 0), reused 5 (delta 0), pack-reused 0
Unpacking objects: 100% (5/5), done.
Tapped 1 formula (30 files, 25.3KB).

treykasada@MacBook-Pro ~> brew install gsts
Updating Homebrew...
==> Installing gsts from ruimarinho/tap
==> Downloading https://github.com/ruimarinho/gsts/archive/v2.4.3.tar.gz
Already downloaded: /Users/treykasada/Library/Caches/Homebrew/downloads/c777c13000c7204d15ee9240640e6af1d279ad6abdbbd302a34b2c42e3cfb889--gsts-2.4.3.tar.gz
==> npm install -ddd --global --build-from-source --cache=/Users/treykasada/Library/Caches/Homebrew/npm_cache --prefix=/usr/local/Cellar/gsts/2.4.3/libexec /private/t
Error: Failed changing dylib ID of /usr/local/Cellar/gsts/2.4.3/libexec/lib/node_modules/gsts/node_modules/puppeteer/.local-chromium/mac-722234/chrome-mac/Chromium.app/Contents/Frameworks/Chromium Framework.framework/Versions/80.0.3987.0/Chromium Framework
  from @executable_path/../Frameworks/Chromium Framework.framework/Versions/80.0.3987.0/Chromium Framework
    to /usr/local/opt/gsts/libexec/lib/node_modules/gsts/node_modules/puppeteer/.local-chromium/mac-722234/chrome-mac/Chromium.app/Contents/Frameworks/Chromium Framework.framework/Versions/80.0.3987.0/Chromium Framework
Error: Failed to fix install linkage
The formula built, but you may encounter issues using it or linking other
formulae against it.
==> Summary
🍺  /usr/local/Cellar/gsts/2.4.3: 4,326 files, 327.6MB, built in 32 seconds

treykasada@MacBook-Pro ~ [1]> which gsts
/usr/local/bin/gsts
```

As you can see, the `brew tap` works fine, but the `brew install` seems to indicate problems during installation. Interestingly though, the `gsts` executable _is present_ after the install, and the scripts I use it in all seem to work in spite of the errors.

Not sure if this is a bug or something cooked with my system, but figured I should raise it anyway. :slightly_smiling_face:

## Component versions
**Homebrew version**: `2.6.2`
**homebrew-core gitref**: `7ae83` (last commit 2020-12-16)
**homebrew-cask gitref**: `81bd71` (last commit 2020-12-16)
**macOS version**: `Big Sur 11.1`
**Shell**: `Fish 3.1.2`

**Comment by pete-leese** [comment]
I also have the same problem - Again running Big Sur 11.1

**Comment by ruimarinho** [comment]
Seems like this is limitation of the version of puppeteer shipped with gsts (2.x). Published 2.4.4 with puppeteer 5.x and the issue is no longer present,  but session persistence isn't working properly. 

Today I will publish the playwright based version which looks much, much better!

**Comment by ruimarinho** [comment]
If you'd like to upgrade to 2.4.4, `brew upgrade gsts`, but this error is harmless.
